packages/reakit/src/Popover/PopoverState.ts

```diff
@@ -100,7 +100,11 @@ export type PopoverInitialState = DialogInitialState &
      */
     unstable_inner?: boolean;
     /**
-     * Offset between the reference and the popover.
+     * Offset between the reference and the popover: [main axis, alt axis]. Should not be combined with `gutter`.
```

Comment:
Technically offset and gutter can be combined, but an explicit offset will overwrite the gutter. Do you think this comment is appropriate, or should it be rephrased / removed?

Reply:
It's fine for now! We can think of a better way to handle this before it becomes stable.